src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -2714,14 +2714,13 @@ def ptq(
             verbose=get_param(calib_params, "verbose"),
         )
         q_util.register_skip_quantization(layer_names=get_param(selective_quantizer_params, "skip_modules"))
-        q_util.quantize_module(model)
 
         model = ptq(
             model,
             selective_quantizer=q_util,
-            calib_loader=calib_loader,
+            calibration_loader=calib_loader,
             calibration_method=get_param(calib_params, "histogram_calib_method"),
-            calibration_batches=get_param(calib_params, "num_calib_batches") or len(calib_loader),
+            calibration_batches=get_param(calib_params, "num_calib_batches") or max(1, int(512 // calib_loader.batch_size)),
```

Comment:
Perhpas its a good idea to actually have 16 is the default here as well? WDYT ?

Reply:
Basically there's some sort of a mix between samples and batches.
Eventually, you want to see enough samples (a reasonable number can be 512), so you calculate the batches you need to see this number of samples.
For 16 batches it becomes dependent on the batch size and not sample side.
It would be better to standardize it (in terms of: passing num samples and deducing batches, and not a mix).